text/head.rs

```diff
@@ -10,70 +10,130 @@
 use clap::Parser;
 use gettextrs::{bind_textdomain_codeset, setlocale, textdomain, LocaleCategory};
 use plib::PROJECT_NAME;
-use std::io::{self, Read, Write};
+use std::error::Error;
+use std::io::{self, Read, StdoutLock, Write};
 use std::path::PathBuf;
 
-/// head - copy the first part of files
+const N_C_GROUP: &str = "N_C_GROUP";
+
+/// head - copy the first part of files.
+/// If neither -n nor -c are specified, copies the first 10 lines of each file (-n 10).
 #[derive(Parser)]
 #[command(version, about)]
 struct Args {
-    /// The first <N> lines of each input file shall be copied to standard output.
-    #[arg(short, default_value_t = 10, value_parser = clap::value_parser!(u64).range(1..))]
-    n: u64,
+    /// The first <N> lines of each input file shall be copied to standard output (mutually exclusive with -c)
+    #[arg(long = "lines", short, value_parser = clap::value_parser!(usize), group = N_C_GROUP)]
+    n: Option<usize>,
+
+    // Note: -c was added to POSIX in POSIX.1-2024, but has been supported on most platforms since the late 1990s
+    // https://pubs.opengroup.org/onlinepubs/9799919799/utilities/head.html
+    //
+    /// The first <N> bytes of each input file shall be copied to standard output (mutually exclusive with -n)
+    #[arg(long = "bytes", short, value_parser = clap::value_parser!(usize), group = N_C_GROUP)]
+    c: Option<usize>,
```

Comment:
Use a better name than `c` for this arg, and use `short =` to set the short option to c.